otp.sh

```diff
@@ -19,13 +21,25 @@ for dir in "$idir"/lib/*/include "$idir"/bootstrap/lib/*/include; do
     includes="$includes -I $dir"
 done
 
+for dir in `find $idir/lib/*/src -name "*.hrl"` `find $idir/bootstrap/lib/*/src -name "*.hrl"`; do
```

Comment:
Some `.hrl` files are not actually in the `includes` folder but under the `src` folder of some applications.